vip-dashboard/components/widget-welcome.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+function render_vip_dashboard_widget_welcome() {
+	?>
+	<div class="widget">
+		<h2 class="widget__title">Welcome to WordPress VIP</h2>
+
+		<p>WordPress VIP is a partnership between Automattic and the most high-profile, innovative and smart WordPress websites out there. We’re excited to have you here.</p>
+
+		<h3 class="widget__subtitle">Helpful Links</h3>
+
+		<div class="widget__col-2">
+			<ul class="widget__list">
+				<li>
+					<a href="https://lobby.vip.wordpress.com/" target="_blank">VIP Lobby</a>
+					<span>Important service updates</span>
+				</li>
+				<li>
+					<a href="https://docs.wpvip.com/" target="_blank">VIP Documentation</a>
+					<span>Launching and developing with VIP</span>
+				</li>
+				<li>
+					<a href="https://wordpressvip.zendesk.com/" target="_blank">VIP Support Portal</a>
+					<span>Your organization’s tickets</span>
+				</li>
+				<li>
+					<a href="https://docs.wpvip.com/technical-references/vip-support/general-ticket-guidelines/" target="_blank">Ticket guidelines</a>
+					<span>How to open the perfect ticket</span>
+				</li>
+			</ul>
+		</div>
+
+		<div class="widget__col-2">
+			<ul class="widget__list">
+				<li>
+					<a href="https://docs.wpvip.com/how-tos/launch-a-site-with-vip/" target="_blank">Guidebook: Launching with VIP</a>
+					<span>Steps to launch</span>
+				</li>
+				<li>
+					<a href="https://wpvip.com/documentation/developing-with-vip/welcome-to-vip-development/" target="_blank">Guidebook: Developing with VIP</a>
```

Comment:
Change URL to https://docs.wpvip.com/technical-references/development-workflow/. 

Perhaps we can also change the copy to something like "Development Workflow on VIP"?